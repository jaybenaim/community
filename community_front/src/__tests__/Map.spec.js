import React from "react";
import { create } from "react-test-renderer";

describe("Map Component Renders", () => {
  test("Map component should render", () => {
    const map = create(
      <section style={{ height: "100vh", width: "100%" }}></section>
    );
    expect(map.toJSON()).toMatchSnapshot();
  });
});
