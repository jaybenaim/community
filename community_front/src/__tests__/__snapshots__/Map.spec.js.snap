// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map Component Renders Map component should render 1`] = `
<section
  style={
    Object {
      "height": "100vh",
      "width": "100%",
    }
  }
/>
`;
